apps/web/components/Organization/Wrapper/Themes/FiskistofaTheme/FiskistofaDefaultHeader.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react'
+
+import { ResponsiveSpace } from '@island.is/island-ui/core'
+import { theme } from '@island.is/island-ui/theme'
+import { DefaultHeader } from '@island.is/web/components'
+import { OrganizationPage } from '@island.is/web/graphql/schema'
+import { useLinkResolver } from '@island.is/web/hooks/useLinkResolver'
+import { useWindowSize } from '@island.is/web/hooks/useViewport'
+
+import * as styles from './FiskistofaDefaultHeader.css'
+
+interface HeaderProps {
+  organizationPage: OrganizationPage
+  logoAltText: string
+  isSubpage: boolean
+}
+
+const FiskistofaDefaultHeader: React.FC<
+  React.PropsWithChildren<HeaderProps>
+> = ({ organizationPage, logoAltText, isSubpage }) => {
+  const { linkResolver } = useLinkResolver()
+
+  const { width } = useWindowSize()
+
+  const themeProp = organizationPage.themeProperties
+
+  return (
+    <div
+      style={{
+        background:
+          (width > theme.breakpoints.lg && !isSubpage
```

Comment:
I wonder how the initial render looks since using js to detect the size takes some time (initial render will take into account css so if we can do this through that it'd be preferrable but I'll approve)